lib/command.notes.js

```diff
@@ -42,7 +42,7 @@ const handler = ({ base, head, pattern, file, verbose }) => {
     const parsed = parseCommits(logs)
 
     const notes = formatNotes({ parsed, version, previous })
-    writeNotes({ notes, version, previous, file })
+    writeNotes({ notes, version: version.trim(), previous: previous.trim(), file })
```

Comment:
this can be done probably at line 37-38 so this gets nicely applied to both `formatNotes` and `writeNotes`